<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlateformOptionWidget</class>
 <widget class="QWidget" name="PlateformOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_objectNameEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="_circleMovementCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Boucle</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Dimensions :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_widthEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_heightEdit">
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="_showTextureCheckbox">
         <property name="text">
          <string>Afficher texture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_loadTextureButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="_addPointButton">
       <property name="text">
        <string>Ajouter un point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="_pointsTable">
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>Points</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_isMovingCheckbox">
       <property name="text">
        <string>En mouvement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_normalizeDirectionCheckbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Accout</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="_speedLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Vitesse : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="_speedSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
